openroad.bzl

```diff
@@ -405,9 +405,9 @@ def yosys_only_attrs():
         ),
     }
 
-def output_attrs(outputs):
+def input_attrs(outputs):
```

Comment:
Nit: `outputs` could be renamed, but should probably be removed and replaced by `default = {}`?